app/client/views/settings/users/enroll/quickfromDropdown.html

```diff
@@ -0,0 +1,12 @@
+<template name="quickForm_dropdown">
+  <div class="form-group">
+    <label for="">
+      {{_ "editUserGroups-groupsLabel" }}
```

Comment:
While this seems like a generic addon for QuickForm, it is using specific localization labels, etc. 

Lets just re-use the other component, rather than extending AutoForm/QuickForm, since we are planning to deprecate AutoForm during the Vue migration.